7_react/react1/week1/vite-project/src/App.jsx

```diff
@@ -0,0 +1,37 @@
+import IteratingTodoItemsArray from "./components/iteratingTodoItemsArray";
+import "./App.css";
+
+const toDoItems = [
+  {
+    id: "1",
+    description: "Get out of bed",
+    deadline: "Wed Sep 13 2017",
+  },
+
+  {
+    id: "2",
+    description: "Brush teeth",
+    deadline: "Thu Sep 14 2017",
+  },
+
+  {
+    id: "3",
+    description: "Eat breakfast",
+    deadline: "Fri Sep 15 2017",
+  },
+];
+
+function App() {
+  return (
+    <>
+      <div className="todo-list">
+        <h3 class="todo-header">Todo List</h3>
```

Comment:
invalid property name, it should be `className` instead

Reply:
Hey @BrantalikP 
Do you mean this part? `<h3 class="todo-header">Todo List</h3>`
Could you please tell me although it is 'class', why it works properly in the css file here? :)
```
.todo-header {
  text-align: center;
  color: brown;
}
```

Reply:
![image](https://github.com/HackYourFuture-CPH/class27-homework/assets/141966593/88102977-5b2c-4c85-b62f-bcbbc0be37cb)